packages/sign-plugin/src/utils/getCreatePluginVersion.ts

```diff
@@ -0,0 +1,18 @@
+import { existsSync, readFileSync } from 'node:fs';
+import { resolve } from 'node:path';
+
+export const getCreatePluginVersion = (distDir: string) => {
```

Comment:
@jackw not sure how we can get the `.cprc.json` when custom dist directory is passed.
```
npx @grafana/sign-plugin@latest --distDir path/to/directory

```

Reply:
Do we need to know where the dist directory is? Generally the `./.config/.cprc.json` is in the root of the project where the command is run. Would `process.cwd()` not work to get the location of the `./.config/.cprc.json` file?

Reply:
My understanding is that `dist/` is usually inside the plugin's root directory where the `.config/.cprc.json` is located, thus one up `../` would be the plugin's root, hopefully. If they run the script outside plugin's root directory and passed in `distDir` directory is not in the project's root, finding `.cprc.json` somewhat impossible.